external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -824,7 +824,13 @@ fn parse_name_access_chain_<'a, F: Fn() -> &'a str>(
             context.tokens.start_loc(),
             " after an address in a module access chain",
         )?;
-        let name = parse_identifier(context)?;
+        let name = match parse_identifier(context) {
+            Ok(ident) => ident,
+            Err(diag) => {
+                context.env.add_diag(*diag);
```

Comment:
Shouldn't we set `is_incomplete` here?

Reply:
I (obviously...) intended to add it, but I ran the test and got distracted by it more or less working as is (without the flag being used at all). The difference now is that we do pick up a different error code that gets created when `is_incomplete` is `true` (and have a different diag as a result)